03-vectors/vectors-practice-2.Rmd

```diff
@@ -62,6 +76,11 @@ draw(p1x, p1y)
 
 you will see Calvin and his mirror reflection, except that there is a problem with the data: some garbage points are mixed in with the correct data, resulting in a lot of spikes that reach reach between 75 and 100 in the y direction. Please fix this problem, so that drawing `p1x` with `p1y` results in a clean rendering of Calvin and his reflection.
 
+```{r fixcalvin}
+garbageIndices = which(p1y>75 & p1y <100)
+draw(p1x[-garbageIndices], p1y[-garbageIndices])
+```
```

Comment:
Using the which function to identify indices of points outside of the parameters and then drawing p1x, p1y without those is really smart and I wish I had been thinking about dealing with indices in this way when I was working on this problem. Nice way to use minimal code to get this done.